### ISSUE TITLE
Every time I start Gifox I must reset the output folder, /tmp

### ISSUE THREAD

**Issue by acolombi** [Gifox/Tracker#181]
I like to use /tmp as my output folder for Gifox on my Mac, 10.15.7 (Catalina). But when I do that I need to rebind the folder after any time I restart my computer (and hence Gifox).

**Comment by ianbytchek** [comment]
Interesting. I'm guessing you're using Gifox version form the Mac App Store?

**Comment by acolombi** [comment]
You are correct, I have Gifox 2 from the Mac Store. I've also noticed that even tho Gifox is in my Login Items, it doesn't load when I reboot. Maybe related?

**Comment by acolombi** [comment]
any update on this? still plagues my workflow

**Comment by jordan-brough** [comment]
If I try to specify `/private/tmp/` as my folder, it doesn't even appear to work at all. I select `/tmp` but then the UI doesn't reflect it.
![2021-02-26 12 17 24](https://user-images.githubusercontent.com/23050/109344958-acec6600-782c-11eb-9143-7907f33f39cc.gif)

**Comment by acolombi** [comment]
Yeah. I think that started failing in the latest version of macOS, BigSur. Now I output to my home directory, which is an annoyance, because I really don't mind losing these GIFs after reboot.
